transformations/question_neural_paraphaser/README.md

```diff
@@ -0,0 +1,38 @@
+# Dialog Rephrasing transformation
+This perturbation uses a model to rephrase a sentence in a natural manner and was proposed for paraphrasing questions.
+
+
+Author name: Pierre Colombo and Emile Chapuis
+Author email: colombo.pierre@gmail.com and emile.chapuis@gmail.com
+Author Affiliation: Telecom Paris
+
+## What type of a transformation is this?
+This transformation uses a neural model finetuned for question rephrasing. 
+
+## What tasks does it intend to benefit?
+This perturbation would benefit all tasks which have a sentence/paragraph/document as input like text classification, 
+text generation, etc. It has been proposed for question paraphrasing.
+
```

Comment:
If you could also do the [robustness evaluation](https://github.com/GEM-benchmark/NL-Augmenter/tree/main/evaluation) like other PRs that'd be great.